clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -26,6 +30,13 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
   cir::PointerType getVoidPtrTy() {
     return getPointerTo(cir::VoidType::get(getContext()));
   }
+
+  mlir::TypedAttr getConstPtrAttr(mlir::Type t, int64_t v) {
```

Comment:
Could we do a better variable name than `v`?  I presume it means `value`?  But that is really only something I can guess at.

Reply:
Done.

I am still calibrating when to use the same names that are in the ClangIR incubator and when to choose better names.